company.json

```diff
@@ -13,10 +13,10 @@
+  "summary_en": "We are a digitalisation partner for healthcare institutions. We communicate directly with healthcare experts to identify the most pressing challenges. We create solutions iteratively. We focus on developing a first prototype quickly, which we evaluate in close collaboration with healthcare professionals. Each iteration then brings a deeper understanding of the problem as well as more specific requirements for improvements. We are not bound to a specific technology because we believe it is important to adapt technology to the solution, not the other way around. The typical result is a web/mobile application or an extension of an existing tool.",
   "summary_de": "Wir sind ein Startup, das sich auf die Entwicklung innovativer IT-Lösungen im Gesundheitswesen orientiert. Wir können unsere Projekte vollständig von der Grundidee und Problemanalyse bis zum Einsetzen in Produktionsumgebung mit voller Kundenunterstützung liefern. Wir nutzen die künstliche Intelligenz, das maschinelle Lernen und die Optimierung in vollem Umfang, aber wir können auch die Entwicklung von Web- oder Mobile Apps problemlos besorgen. Ein Schlüsselfaktor ist auch unsere perfekte Kenntnis des Gesundheitswesens und die Fähigkeit, medizinische Probleme gut zu verstehen. Wir sind ein Team von begeisterten Menschen, die bereit sind, jedes interessante medizinische Problem zu lösen und die medizinische Welt zu verbessern. Derzeit arbeiten wir an mehreren solchen Projekten, von denen einige schon in Krankenhäusern eingesetzt sind.",
-  "mission_cs": "Naší vizí jsou lidé s delšími, zdravějšími a šťastnějšími životy. Ve zdravotnictví vidíme obrovský potenciál k vylepšení jak medicínských, tak administrativních procesů pomocí digitálních řešení. V Mild Blue umíme tento potenciál využít díky kombinovaným znalostem nejmodernějších technologií i medicíny. Spolupracujeme s výzkumnými organizacemi, abychom výsledky bádání aplikovali přímo do praxe a zároveň obohacovali samotný výzkum. Věříme, že to je pro nás nejlepší způsob, jak můžeme mít pozitivní dopad na společnost.",
+  "mission_cs": "Držíme krok s nejmodernějšími technologiemi ale rozumíme specifikám zdravotnictví. Ve zdravotnictví vidíme obrovský potenciál k vylepšení jak medicínských, tak administrativních procesů pomocí digitálních řešení. V Mild Blue umíme tento potenciál využít díky kombinovaným znalostem nejmodernějších technologií i medicíny. Spolupracujeme s výzkumnými organizacemi, abychom výsledky bádání aplikovali přímo do praxe a zároveň obohacovali samotný výzkum. Věříme, že to je pro nás nejlepší způsob, jak můžeme mít pozitivní dopad na společnost.",
```

Comment:
tohle mi zase nesedi k titulku "cemu verime" ale veta to neni spatna.
Co hned ta druha veta vize? Chceme žít ve světě, kde se nejnovější technologie efektivně využívají k neustálému zlepšování úrovně zdravotní péče.